docs/products/cassandra/howto/list-cross-cluster-replication.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: Cross-cluster replication
+pro: true
```

Comment:
Already raised it on slack, but just for being extra sure! Pro Platform is now Aiven Entreprise